pkg/autoupdate/autoupdate.go

```diff
@@ -64,7 +65,7 @@ func NewUpdater(binaryPath, rootDirectory string, logger log.Logger, opts ...Upd
 	strippedBinaryName := strings.TrimSuffix(binaryName, ".exe")
 	tufRepoPath := filepath.Join(rootDirectory, fmt.Sprintf("%s-tuf", strippedBinaryName))
 	stagingPath := filepath.Join(filepath.Dir(binaryPath), fmt.Sprintf("%s-staging", binaryName))
-	gun := fmt.Sprintf("kolide/%s", strippedBinaryName)
+	gun := path.Join(DefaultNotaryPrefix, strippedBinaryName)
```

Comment:
I know this is the existing code, but it seems cleaner to skip the `gun` assignment, and just stick the `path.Join(...)` into the `tuf.Settings` struct a couple lines down